tools/topology/topology2/production/tplg-targets-hda-generic.cmake

```diff
@@ -6,4 +6,13 @@ DEEPBUFFER_D0I3_COMPATIBLE=true"
 # HDA topology with mixer-based pipelines for HDA and passthrough pipelines for HDMI
 "sof-hda-generic\;sof-hda-generic\;HDA_CONFIG=mix,DEEPBUFFER_FW_DMA_MS=100,\
 DEEPBUFFER_D0I3_COMPATIBLE=true"
+
+# HDA topology with mixer-based pipelines for HDA and
+# passthrough pipelines for HDMI and
+# 2 or 4 DMIC, no NHLT blob included in topology
+"sof-hda-generic\;sof-hda-generic-2ch\;HDA_CONFIG=mix,NUM_DMICS=2,\
```

Comment:
so, at the previous patch the tplg2 build was incomplete?
I know bisecting is not a primary concern, just pointing out.